doc/api/fs.md

```diff
@@ -1524,11 +1524,27 @@ If `options` is a string, then it specifies the encoding. Example:
 ```js
 fs.readFile('/etc/passwd', 'utf8', callback);
 ```
+*Note*: When the given path identifies a directory, the behavior of
+`fs.readFile()` and `fs.readFileSync()` is platform specific.On macOS, Linux
```

Comment:
@adityaanandmc Comma after Linux and *keep* the `and`:

> On macOs, Linux, and Windows, an error will be returned.

That would be a [serial comma](https://en.wikipedia.org/wiki/Serial_comma) and be in accordance with [the style guide for Node.js docs](https://github.com/nodejs/node/blob/master/doc/STYLE_GUIDE.md).